ccpp/data/GFS_typedefs.F90

```diff
@@ -2746,13 +2749,12 @@ subroutine sfcprop_create (Sfcprop, IM, Model)
        !
     end if
 
-    allocate (Sfcprop%rmol   (IM ))
-    allocate (Sfcprop%flhc   (IM ))
-    allocate (Sfcprop%flqc   (IM ))
-    Sfcprop%rmol        = clear_val
-    Sfcprop%flhc        = clear_val
-    Sfcprop%flqc        = clear_val
-
+       allocate (Sfcprop%rmol   (IM ))
```

Comment:
Can you undo this whitespace change? These statements should be at the same indentation as the `end if` above for clarity.